test/integ-new/main_new_test.go

```diff
@@ -303,6 +304,38 @@ func TestCertRotatorKeyRotation(t *testing.T) {
 				3: kmspb.CryptoKeyVersion_ENABLED,
 			})
 	})
+
+	t.Run("invalid_primary", func(t *testing.T) {
+		// Set primary to a version that doesn't exist
+		if err := jvscrypto.SetPrimary(ctx, kmsClient, keyResouceName, keyResouceName+"/cryptoKeyVersions/99"); err != nil {
```

Comment:
Is this actually possible in the upstream API? I don't think KMS will let you set the primary version to a non-existent thing.

Reply:
I think the primary is just a label, we can set it like this: https://github.com/abcxyz/jvs/blob/main/.github/workflows/ci-new.yml#L183

Reply:
Ah I see - we're not setting the "primary" field because it's an asymmetric key?

Reply:
Yes